docs/4devs.md

```diff
@@ -30,3 +30,8 @@ For _very_ rough statistics, open devTools -> Network and look at the line that
 ## Subscriptions and queries
 
 For the performance of the app, it is important that we limit the amount of queries and subscriptions that alchemy creates when loading a page.
+
+## Coding conventions
+
+- As an ongoing process of adopting language localization, any new static text should be added to a translation file and used via `i18next` internationalization-framework.
+Translation files location is under `src/assets/locales/`.
```

Comment:
Can you give a code example here?

Reply:
I think in this case it's really easy to track an existing code and understand how to work with this even for someone new.
This document should stay as simple as possible.

Reply:
You are assuming the user knows what you are asking them to do.  A simple one-line piece of code example using `i18next.t("key")` does not add significant complexity to this document and makes clear what we are expecting the user to do, saving them time and worry.  What cost is there in doing that?

Reply:
Done.